### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by Tiago-Lelinski-Marin**
In this pull request, my partner and I added the form and styled it with CSS. I was also working on desktop responsiveness, but that's part of the next submission. 
So feel free to give me any feedback on the responsiveness, but please do not consider it when giving the approval on this PR.

Here's the live demo in case you need it:
https://tiago-lelinski-marin.github.io/myPortfolio/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,72 +183,120 @@ <h3 class="card-h">Profesional Art <br> Printing Data</h3>
...
+    </div>
+    <section class="contact-me">
+        <h3>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h3>
+        <div class="form">
+            <form action="https://formspree.io/f/mrgjjnpd" method="post"></form>
+            <input id="name" type="text" placeholder="Full Name" maxlength="30" required>
+            <input id="email" type="email" name="email" placeholder="Email address">
+            <textarea id="message" name="message" placeholder="Enter text here" maxlength="500"></textarea>
+            <input id="submit" type="submit" value="Get in touch">
```
Comment (hiromataba) [inline index.html:267]: - [ ] Well done on the design 👍 , however, your form is not working now and the reason is that you forgot to wrap all your inputs inside od the form, please put the closing tag of your form after the submit input and your form will work as expected

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Tiago-Lelinski-Marin and @basitali111    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Tiago-Lelinski-Marin, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors.
✅ Good implementing changes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Code Review by @omarsalem7

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tiago-Lelinski-Marin** [comment]
Hi @omarsalem7 how are you doing?

I only see the "Review Ppending" on my Dashboard, it should be "Submit", shouldn't it?

image

**Comment by Tiago-Lelinski-Marin** [comment]
Sorry, I deleted my previous comment, I guess I just needed to refresh my browser.

**Pull request merged**